java-lib/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please illustrate how this file comes into effect? In `BUILD` or `WORKSPACE` files I didn't see any usage of this pom file.

Reply:
This dockerfile is not needed to build/run Clink. Its purpose is setup IDE (e.g. Intellij) and make it easy for developers to use IntellIj to develop Java program in Clink. Does this make sense?

Reply:
Yes it makes sense.